### PR TITLE
Update julia.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Made an update so that those looking for other support resources know where to find it.

### CODE REVIEW

**Comment by logankilpatrick** [comment]
@plaindocs Let me know if you ave any suggested changes!

**Comment by logankilpatrick** [comment]
Fixed a deprecated link and updated the disclaimer on that code so people know it will only build on Linux and not MacOS.

File: user/languages/julia.md [hunk 1]
```diff
@@ -15,9 +15,11 @@ This guide covers build environment and configuration topics specific to
 
 Travis CI support for Julia is contributed by the community and may be removed
 or altered at any time. If you run into any problems, please report them in the
-[Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues/new?labels=julia)
+[Travis CI Julia Community Forums](https://travis-ci.community/c/languages/julia)
 and cc [@ararslan](https://github.com/ararslan), [@staticfloat](https://github.com/staticfloat), and [@StefanKarpinski](https://github.com/StefanKarpinski).
 
+General support for Julia on Travis CI can be found on [Travis Community](https://travis-ci.community/c/languages/julia) or on the [Julia Lang Slack Channel](https://julialang.slack.com) under the __#testing__ channel.
```
Comment (Pezi777) [inline user/languages/julia.md]:
```suggestion
For general Julia support on Travis CI go to the [Travis Community](https://travis-ci.community/c/languages/julia) or [Julia Lang Slack Channel](https://julialang.slack.com) in the __#testing__ channel.
```

File: user/languages/julia.md [hunk 2]
```diff
@@ -92,7 +94,7 @@ to manually install any dependencies that are not handled by `deps/build.jl`,
 it is possible to specify a custom dependency installation command as described
 in the [general build configuration](/user/customizing-the-build/) guide.
 
-In a rare case, you may need to clone a private repo if it is a depenedancy of the repo you are trying to test. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below to see how that is done: 
+In a rare case, you may need to clone a private repo if it is a depenedancy of the repo you are trying to test. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done:
```
Comment (Pezi777) [inline user/languages/julia.md]:
```suggestion
In a rare case, you may need to clone a private repo, if you are trying to test a dependency of the repo. To add a private repo, check out the link here: [Private Dependencies](/user/private-dependencies/).  Once you have the repo added, you will need to copy it to your julia folder and then run the default build script.  Check out the script below for __Linux__ to see how that is done: 
```

Comment (logankilpatrick) [reply user/languages/julia.md]: @Pezi777 In this example, you are not testing a dependancy. You are testing the repo!

Comment (logankilpatrick) [reply user/languages/julia.md]: Therefore I stand by the original language I used.

Comment (logankilpatrick) [reply user/languages/julia.md]: @Pezi777 any thoughts?

Comment (Pezi777) [reply user/languages/julia.md]: we can keep the wording, but it is nevertheless "dependency" and not "depenedancy" - right?

Comment (logankilpatrick) [reply user/languages/julia.md]: That’s correct! My mistake on the spelling error.

**Comment by logankilpatrick** [comment]
Fixed the spelling in the latest commit @Pezi777 !

**Pull request merged**